server/routes/index.js

```diff
@@ -0,0 +1,7 @@
+const device = require('./device');
```

Comment:
### Nitpick

Let's keep our naming consistent and descriptive. For browsers, you imported it as `browserRoutes` but called this `device`. `deviceRoutes` is better suited and the others should follow that convention.